.jenkins/pytorch/test.sh

```diff
@@ -519,11 +519,19 @@ fi
 
 if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
   pushd test
-  echo "Generating XML coverage report"
-  time python -mcoverage xml
+  if [[ -f "./coverage.xml" ]]; then
+    echo "Generating XML coverage report"
+    time python -mcoverage xml
+  else
+    echo "No python coverage report found. Skipping"
+  fi
   popd
   pushd build
-  echo "Generating lcov coverage report for C++ sources"
-  time lcov --capture --directory . --output-file coverage.info
+  if [[ -f "./coverage.info" ]]; then
```

Comment:
```suggestion
  if [[ -f "./**/*.gcda" && -f "./**/*.gcno" ]]; then
```
It would be this, but I don't actually like this because I would want to know when these files are missing (versus it silently failing).

Reply:
won't the coverage report have a significant drop to 0% if file were missing?